src/main/java/org/fenixedu/academic/domain/student/curriculum/calculator/ConclusionGradeCalculatorResultsDTO.java

```diff
@@ -8,10 +8,19 @@ public class ConclusionGradeCalculatorResultsDTO {
     private final Grade intermediateRoundedGrade;
     private final Grade finalGrade;
 
+    private String htmlDescription;
+
     public ConclusionGradeCalculatorResultsDTO(Grade unroundedGrade, Grade intermediateRoundedGrade, Grade finalGrade) {
         this.unroundedGrade = unroundedGrade;
         this.intermediateRoundedGrade = intermediateRoundedGrade;
         this.finalGrade = finalGrade;
+        this.htmlDescription = null;
+    }
+
+    public ConclusionGradeCalculatorResultsDTO(Grade unroundedGrade, Grade intermediateRoundedGrade, Grade finalGrade,
```

Comment:
being the description an optional field, its better to have a new constructor, or a setter method?